components/integration/replace-camel/code/camel-bbe/for-integrations.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Language built for integrations'
+description: 'Ballerina excels with its first-class services, listeners, and clients, making integration capabilities central to its design. Build scalable integrations effortlessly, thanks to its strong type system and native support for JSON and XML. Experience the simplicity and readability of Ballerina for seamless integration development.'
+url: 'https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-zapier-samples/gdrive-new-event-to-slack-message'
+---
+```
+import ballerinax/trigger.google.drive;
```

Comment:
I think we better use a different sample with a service, client and XML,json usage?